include/net/ieee802154_mgmt.h

```diff
@@ -0,0 +1,233 @@
+/*
+ * Copyright (c) 2017 Intel Corporation.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+/**
+ * @file
+ * @brief IEEE 802.15.4 Management interface public header
+ */
+
+#ifndef __IEEE802154_MGMT_H__
+#define __IEEE802154_MGMT_H__
+
+#include <net/net_mgmt.h>
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief IEEE 802.15.4 library
+ * @defgroup ieee802154 IEEE 802.15.4 Library
```

Comment:
defgroup ``iee802154`` is also still defined in iee802154.h  I suspect you should delete that defgroup or change that one to an addtogroup to avoid confusion and a potential error.